src/RepoMerger.js

```diff
@@ -96,6 +96,10 @@ class RepoMerger {
 
     _plan(ms) {
         assert(ms > 0);
+        // obey node.js setTimeout() limits
+        const maxMs = Math.pow(2, 31) - 1;
+        ms = ms < maxMs ? ms : maxMs;
```

Comment:
```suggestion
        const ms = Math.min(requestedMs, maxMs);
```

and rename the parameter to requestedMs, of course.

Reply:
Done.